platform/linux-generic/arch/aarch64/odp/api/cpu_arch.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2016, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef ODP_PLAT_CPU_ARCH_H_
+#define ODP_PLAT_CPU_ARCH_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define _ODP_CACHE_LINE_SIZE 64
+
+static inline void odp_cpu_pause(void)
+{
+	/* YIELD hints the CPU to switch to another thread if possible
+	 * and executes as a NOP otherwise.
+	 * ISB flushes the pipeline, then restarts. This is guaranteed to
+	 * stall the CPU a number of cycles.
+	 */
+	__asm volatile("isb" ::: "memory");
```

Comment:
that looks like has to be __asm__ __volatile__ to be consistent to other calls. That doc https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html says about __asm__ but I did not see doc which talked about __volatile__ instead of volatile. Do you know where to read details?

Reply:
This is just a copy of arm's file. We can change both of them, but it's a separate doing.